tests/core/test_config.py

```diff
@@ -98,3 +100,27 @@ def load_cdns(cls, _db, config_instance=None):
         assert new_db != self._db
         assert isinstance(new_db, Session)
         assert None == none
+
+    def test_fcm_credentials_file(self):
+        env_var = os.environ.pop(
```

Comment:
Should we use `monkeypatch`'s `setenv` and `delenv` here to isolate changes to the environment to make them safer?

Reply:
This is like the most basic method of testing and I don't know why I did what I did. Updated.